### PR TITLE
[7.4-only] LPS-121968 Migrate v2 usages of clay:vertical-card in site-admin-web

### PR DESCRIPTION
**Pull request by julien**
As seen with @p2kmgcl this is a good candidate for `<clay:navigation-card />` so we suggest changing it.
cc @carloslancha @markocikos

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

File: modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/servlet/taglib/clay/SelectSiteInitializerVerticalCard.java [hunk 1]
```diff
@@ -48,6 +48,11 @@ public SelectSiteInitializerVerticalCard(
 		_httpServletRequest = PortalUtil.getHttpServletRequest(renderRequest);
 	}
 
+	@Override
+	public String getCssClass() {
+		return "add-site-action-card mb-0";
```
Comment (carloslancha) [inline modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/servlet/taglib/clay/SelectSiteInitializerVerticalCard.java:53]: I think we're missing original classes defined in `getElementClasses` here

Comment (julien) [reply modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/servlet/taglib/clay/SelectSiteInitializerVerticalCard.java:53]: I'm not 100% sure about that, so it's a good thing you mention it, before this change we had 
an `elementClasses` attribute and the classes defined in `getElementClasses`

Comment (carloslancha) [reply modules/apps/site/site-admin-web/src/main/java/com/liferay/site/admin/web/internal/servlet/taglib/clay/SelectSiteInitializerVerticalCard.java:53]: I see that `elementClasses` where defined both via attributes and via model. Taking  a look to soy tag version looks like attributes had preference, so if `elementClasses` where passed as an attribute this `getElementClasses` method were ignored. Please check if I'm right and, if so, forget about my comment ('cause `getElementClasses` was doing nothing :))

**Review by carloslancha — COMMENTED** [review]
Don't we need to migrate `getData`?

**Comment by julien** [comment]
@carloslancha that's also what we saw with @p2kmgcl so I think this is OK - but I appreciate the review :+1: better safe than sorry

**Comment by julien** [comment]
ci:stop

**Comment by julien** [comment]
ci:test:relevant

**Review by carloslancha — APPROVED** [review]

**Comment by p2kmgcl** [comment]
ci:forward

**Pull request closed without merging**